docs/gen-docs/kyma_alpha.md

```diff
@@ -24,4 +24,5 @@ Use this command to utilize the Kyma CLI in alpha testing stage.
 * [kyma alpha deploy](#kyma-alpha-deploy-kyma-alpha-deploy)	 - Deploys Kyma on a running Kubernetes cluster.
 * [kyma alpha provision](#kyma-alpha-provision-kyma-alpha-provision)	 - Provisions a cluster for Kyma installation.
 * [kyma alpha uninstall](#kyma-alpha-uninstall-kyma-alpha-uninstall)	 - Uninstalls Kyma from a running Kubernetes cluster.
+* [kyma alpha version](#kyma-alpha-version-kyma-alpha-version)	 - Displays the version of Kyma CLI and the connected Kyma cluster.
```

Comment:
This should be clear, because the command says `version`, but to be 100% safe, 
I would go with 
```suggestion
* [kyma alpha version](#kyma-alpha-version-kyma-alpha-version)	 - Displays the version of Kyma CLI and of the connected Kyma cluster.
```
or 
`the versions of Kyma CLI and the connected Kyma cluster.` to avoid the ambiguity that it displays the cluster to you (points you to the cluster or something).

Reply:
Yeah, I agree, makes sense